Exec/RegTests/FlameSheet/GNUmakefile

```diff
@@ -71,6 +71,10 @@ CEXE_headers +=
 FEXE_headers += 
 
 include $(PELELM_HOME)/Tools/Make/Make.PeleLM
+
+ifeq ($(USE_CUDA),TRUE)
+CXXFLAGS += -Xptxas --disable-optimizer-constants
+endif
```

Comment:
Should this rather go into Make.PeleLM?

Reply:
I guess it should yes ! I'll move it in an upcoming PR.